Index: src/com/company/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/company/Main.java	(date 1578056216304)
+++ src/com/company/Main.java	(date 1578056216304)
@@ -0,0 +1,9 @@
+package com.company;
+
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+        System.out.println("pobieram z pliku");
+    }
+}
